src/scripts.js

```diff
@@ -35,15 +37,21 @@ let formActivityFlights = document.querySelector('#flights');
 let submitButton = document.querySelector('.submit-button');
 let xButton = document.querySelector('#remove');
 
-let formSuccessNotification = document.querySelector('.success');
-let formErrorNotification = document.querySelector('.error');
+let formSuccessNotification = document.querySelector('#successNotification');
+let formErrorNotification = document.querySelector('#failureNotification');
 
 window.addEventListener('load', getFetchedUsers);
 hydrationButton.addEventListener('click', showHydrationForm);
 sleepButton.addEventListener('click', showSleepForm);
 activityButton.addEventListener('click', showActivityForm);
 submitButton.addEventListener('click', submitForm);
 xButton.addEventListener('click', hideAllForms);
+baseForm.addEventListener("keydown", function (event) {
+  var invalidCharacters = ["e", "+", "-", "."];
+  if (invalidCharacters.includes(event.key)) {
+    event.preventDefault();
+  }
+});
```

Comment:
we should through this into a normal function so that the event listener isn't using an anonymous function